src/backend/utils/error/elog.c

```diff
@@ -3250,6 +3253,15 @@ send_message_to_frontend(ErrorData *edata)
 			err_sendstring(&msgbuf, edata->funcname);
 		}
 
+		if (sql_dialect == SQL_DIALECT_TSQL)
+		{
+			if (edata->message_id)
```

Comment:
how do we make sure that all clients will be able to handle additional message id in error messgage

Reply:
are we sure that parallel workers have consistent mapping with respect to dialect with leader node? what if error happens inside a procedure executed with PG dialect or vice versa and leader has different dialect?

Reply:
As discussed offline, we can keep this message_id irrespective of dialect.